src/lib/y2storage/disk_analyzer.rb

```diff
@@ -284,34 +295,39 @@ def release_name(filesystem)
       fs.release_name
     end
 
-    # TODO: there is no reason to initally discard usb devices. The behavior is
-    # kept to avoid big changes in release phase.
+    # Finds devices (disk devices or software raids) that are suitable for installing Linux
```

Comment:
NP: I guess it should be "and" -> (disk devices and software raids)

Reply:
Yep, you are right.